_includes/install_from_obs_deb.md

```diff
@@ -28,7 +28,7 @@ You can find available options on the [installation page at OBS](https://softwar
 
 <div class="code_section">
 {% highlight bash %}
-echo "deb http://download.opensuse.org/repositories/devel:/languages:/crystal/{REPOSITORY}/ /" | tee /etc/apt/sources.list.d/crystal.list
+sudo echo "deb http://download.opensuse.org/repositories/devel:/languages:/crystal/{REPOSITORY}/ /" | tee /etc/apt/sources.list.d/crystal.list
```

Comment:
`echo` doesn't need sudo, `tee` does (like below):
```suggestion
echo "deb http://download.opensuse.org/repositories/devel:/languages:/crystal/{REPOSITORY}/ /" | sudo tee /etc/apt/sources.list.d/crystal.list
```

Reply:
Ah, misplaced it! Anyway, thanks for the review, @straight-shoota.